bot/code_review_bot/revisions.py

```diff
@@ -134,20 +152,52 @@ def __str__(self):
         return "Phabricator #{} - {}".format(self.diff_id, self.diff_phid)
 
     @staticmethod
-    def from_try(try_task: dict, phabricator: PhabricatorAPI):
+    def from_try(try_decision_task: dict, phabricator: PhabricatorAPI):
         """
         Load identifiers from Phabricator, using the remote task description
         """
+        logger.info(
+            "Using try decision task", name=try_decision_task["metadata"]["name"]
+        )
+
+        def match_task_env(repository):
```

Comment:
Could we keep matching the task with the Phabricator repository? This way we avoid issues where two repos have the same envs

Reply:
I don't really see how: at that point in the timeline, we only have access to the tasks definition (and the mercurial informations buried in it). before we had access to the specific phabricator build id, which gave me the specific repository. Now that's the opposite

Reply:
Ah because we don't have the URL to get try_task_config.json, that's painful.